aeon/schema/social_03.py

```diff
@@ -6,3 +8,9 @@ class Pose(Stream):
 
     def __init__(self, path):
         super().__init__(_reader.Pose(f"{path}_202_*"))
+
+
+class EnvActiveConfiguration(Stream):
```

Comment:
All looks great, I would just rename this stream to be called `EnvironmentActiveConfiguration` rather than the shorthand version, but not critical either way.

Reply:
Done